fv3core/stencils/saturation_adjustment.py

```diff
@@ -459,9 +390,7 @@ def sublimation(
             src = sink if sink < maxtmp else maxtmp
             src = src if src < tmp / tcp2 else tmp / tcp2
         else:
-            dimtmp = (
-                pt1 - t_sub if (pt1 - t_sub) > 0.0 else 0.0
-            )  # dim(pt1, t_sub) * 0.2 TODO WHY DOES THIS NOT WORK
+            dimtmp = dim(pt1, t_sub)
```

Comment:
should we double check where the * 0.2 comes from in the comment? could be a bug in the comment or in the code

Reply:
In my branch, `dimtmp` gets multiplied by 0.2 when computing `pidep`.  Perhaps one change I can do is to move the multiplication of 0.2 to `dimtmp` and then remove the multiplication of 0.2 from `pidep`.

Reply:
ahh -- ok it's just an outdated comment there, awesome, can leave as is